src/nvim/lua/executor.c

```diff
@@ -1228,8 +1229,10 @@ void ex_luado(exarg_T *const eap)
           new_line_transformed[i] = '\n';
         }
       }
+
       ml_replace(l, (char_u *)new_line_transformed, false);
-      changed_bytes(l, 0);
+      size_t old_line_len = curbuf->deleted_bytes2 - 1;
```

Comment:
Do we really have to use this attribute here? This will make its removal even more difficult, why not just take the old line len before the edits happens?

Reply:
yes, we already have `ml_get_buf(curbuf, l, false)` from above.

Reply:
Yes, that makes sense. I've reverted it to how it was before